metadata-load/src/test/java/uk/ac/ebi/ampt2d/metadata/importer/api/SraObjectsImporterThroughApiTest.java

```diff
@@ -76,12 +77,22 @@ public void setUp() {
 
     @Test
     public void importStudy() throws Exception {
-        // This study contains two analyses and one reference sequence
+        // This study contains two analyses with referenceSequences
         Study study = sraObjectImporter.importStudy("ERP006576");
         assertNotNull(study);
         assertEquals("ERP006576", study.getAccessionVersionId().getAccession());
         assertEquals(LocalDate.of(2014, 8, 4), study.getReleaseDate());
         assertEquals("Sanger Institute Mouse Genomes Project v3", study.getName());
+        assertEquals(2, study.getAnalyses().size());
+
+        study = sraObjectImporter.importStudy("ERP111092");
+        assertNotNull(study);
+        assertEquals("ERP111092", study.getAccessionVersionId().getAccession());
+        assertEquals(LocalDate.of(2018, 9, 28), study.getReleaseDate());
+        assertEquals("Genetic diversity at LPL gene in river buffalo", study.getName());
+
+        //The analysis of the study does not contain a ReferenceSequence so it isn't imported.
```

Comment:
probably a better rewording?
// The analysis associated with the study does not contain any reference sequence; hence analysis isn't imported

Reply:
I guess both means the same if there is any grammatical error, I can change it.

Reply:
I think the simplest description would be: 
```suggestion
        // The analysis in the study is not imported because it is not linked to a ReferenceSequence
```